test/extensions/transport_sockets/tls/test_data/long_validity_key.pem

```diff
@@ -1,27 +1,27 @@
 -----BEGIN RSA PRIVATE KEY-----
-MIIEpQIBAAKCAQEA1O7tGCdCNtPdHmSo+vOv/NeisL96PNQtoxSWUoeKGkFpwNuq
```

Comment:
Did all of these files really change, or did they just get regenerated as a group by running a script? Is it worth reverting the onces that didn't functionally change to make the diff smaller?

Reply:
i added the extensions to the main CA so they all had to be regenerated. Updated to add a new cert instead, which means we don't need to recreate all the certs